src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -259,6 +259,38 @@ public async Task SslStream_StreamToStream_Successive_ClientWrite_WithZeroBytes_
             }
         }
 
+        [Fact]
+        public async Task SslStream_StreamToStream_ZeroByteRead_SucceedsWhenDataAvailable()
+        {
+            using var listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
```

Comment:
You can use TestHelper.GetConnectedTcpStreams()

Reply:
Thanks.  Changed to use that.